### ISSUE TITLE
New license namespace request: ae [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by karsten-klein** [spdx/license-list-XML#1414]
**1.** License Name: metaeffekt
**2.** Short identifier: ae
**3.** License Author or steward: Karsten Klein
**4.** Comments: 
**5.** Standard License Header: 
**6.** License Request Url: http://tools.spdx.org/app/license_requests/143
**7.** URL(s): https://metaeffekt.com/#universe
**8.** OSI Status: 
**9.** Example Projects:

**Comment by jlovejoy** [comment]
hi @karsten-klein - This doesn't appear to be an actual license?  I'm confused... can you explain?

**Comment by karsten-klein** [comment]
I'm confused as well. This is the result from my namespace registration in the SPDX Online Tool:
 
https://tools.spdx.org/app/license_namespace_requests/8/

I yesterday promoted the request (which was already confusing as it said that it will promote this to the licenses list).

I currently would say the SPDX Online Tool may not be fully consistent with what it suggests. You can delete or close the license request. And we can sort out, whether I'm able to register an officially accepted SPDX namespace with the short id 'ae' and full name 'metaeffekt'.

**Comment by seabass-labrax** [comment]
Hmm, looks like this is an oversight relating to how the SPDX Online Tools produces the GitHub issues. I've filed an issue at https://github.com/spdx/spdx-online-tools/issues/337.

**Comment by seabass-labrax** [comment]
> I'm confused as well. This is the result from my namespace registration in the SPDX Online Tool

Hmm, looks like this is an oversight relating to how the SPDX Online Tools produces the GitHub issues. I've filed an issue at https://github.com/spdx/spdx-online-tools/issues/337.

**Comment by karsten-klein** [comment]
Is there any way to proceed with the original license namespace request?

**Comment by zvr** [comment]
@karsten-klein we have to reach consensus how to define and use namespaces first.
https://github.com/spdx/spdx-spec/pull/209 is still work in porgress

**Comment by jlovejoy** [comment]
Thanks @zvr @seabass-labrax - yeah, I noticed this as well and, indeed, there appears to be a bug that submission on the namespace tool also generates an issue here. But more importantly, the namespace proposal never got officially incorporated into the spec, so in that way this tool (which was a GSoC project) got ahead of the proposal! I have raised this with the Steering Committee. (and going to close this issue, as it's not directly related to the SPDX License List!)
